dbt_subprojects/daily_spellbook/models/_sector/airdrops/ethereum/projects/x2y2_ethereum_airdrop_claims.sql

```diff
@@ -18,7 +18,7 @@
 WITH early_price AS (
     SELECT MIN(minute) AS minute
     , MIN_BY(price, minute) AS price
-    FROM {{ source('prices', 'usd') }}
+    FROM prices.usd
```

Comment:
We need to use the {{}} format for sources and refs in the spell.
```suggestion
    FROM {{ source('prices', 'usd') }}
```